arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/README.md

```diff
@@ -0,0 +1,50 @@
+# Create terrain from a local tile package
+
+Set the terrain surface with elevation described by a local tile package.
+
+![Create terrain from a local tile package](create-terrain-from-a-local-tile-package.png)
+
+## Use Case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. The tile package must be a LERC (limited error raster compression) encoded TPK. Details on creating these are in the [ArcGIS Pro documentation](https://pro.arcgis.com/en/pro-app/help/sharing/overview/tile-package.htm).
+
+Terrain can be loaded offline from tile packages (.tpk).
+
+## How it works
+
+1. Create an `ASScene` and add it to a `SceneView`.
```

Comment:
typo: `ASScene` -> `AGSScene`

The class name is `AGSSceneView`. You can look at our API doc if you're not sure - http://morphine.esri.com/api-ref/ios/annotated.html